packages/react-reconciler/src/__tests__/ReactUpdaters-test.internal.js

```diff
@@ -262,6 +262,9 @@ describe('updaters', () => {
     await waitForAll([]);
   });
 
+  // This test should be convertable to createRoot but the allScheduledTypes assertions are no longer the same
+  // So I'm leaving it in legacy mode for now and just disabling if legacy mode is turned off
+  // @gate !disableLegacyMode
```

Comment:
This was my latest finding regarding the change of scheduler types: https://github.com/facebook/react/pull/28330#issuecomment-1967923417

Haven't looked deeper into it.